app/workers/data_update_worker.rb

```diff
@@ -16,10 +14,16 @@ def perform
 
   def run_script(script)
     require script.file_path
+
+    Rails.logger.info("Data update script: #{script.file_name}: running")
```

Comment:
Since our whole team doesn't have access to our logs can we instead send this to Datadog so people can then check there that their scripts ran. 
```ruby
DataDogStatsClient.increment("data_update_scripts.status", tags: ["action:start", "script_name:#{script.file_name}"])
```

Reply:
It's okay if we do both? I can't see DD in the console. I'll add all the DD statements!

Reply:
Both works for me! I wonder if we could patch that stats client to log in development 🤔 That would be slick! brb making a ticket for that lol